webapp/components/wpt-interop.html

```diff
@@ -330,8 +333,8 @@
       return metadata && metadata.test_runs;
     }
 
-    computeInteropNotFound(isLoading, interop) {
-      return !isLoading && !interop;
+    computeInteropNotFound(wasLoading, isLoading, passRateMetadata) {
```

Comment:
Here and below. If we observe nothing but `isLoading`, then the observer function can see previous value and new value. Iff old value was `true` and new value is `false`, we can then manually check `this.passRateMetadata` this obviates the need for `wasLoading`. This also applies to `computeRunsNotFound` in `wpt-results.html`.